include/SW_Control.h

```diff
@@ -32,12 +32,12 @@ void SW_CTL_setup_model(SW_ALL* sw, SW_OUTPUT_POINTERS* SW_OutputPtrs,
                         PATH_INFO* PathInfo, LOG_INFO* LogInfo);
 void SW_CTL_clear_model(Bool full_reset, SW_ALL* sw, PATH_INFO* PathInfo);
 void SW_CTL_init_run(SW_ALL* sw, LOG_INFO* LogInfo);
-void SW_CTL_read_inputs_from_disk(SW_ALL* sw, PATH_INFO* PathInfo,

```

Comment:
Can you please update both branches with v7.1.0? -- currently `feature_nc_output` does not have the `InFiles` commit and all the diffs from that commit show up here as well